Servers/UI/OJS.Servers.Ui/ClientApp/src/components/contests/contest/Contest.tsx

```diff
@@ -98,6 +101,23 @@ const Contest = () => {
         [],
     );
 
+    const renderParticipants = useCallback(
+        (participants: number, isActive: boolean) => (
+            <span>
+                {isActive
+                    ? 'Active'
+                    : 'Total'}
+                {' '}
+                Participitants:
+                {' '}
+                <Text type={TextType.Bold}>
+                    {participants.toString()}
```

Comment:
Is `toString` necessary here?

Reply:
It is required because of the following error: "Texts must have only `text` or `children` at Text"